hw1/accessTernary.js

```diff
@@ -0,0 +1,31 @@
+function hasAccessTern(user) {
+    return (user.age < 18 || user.age > 35) ? false :
+        (user.isAdmin ) ? true : user.paid && !user.blocked && !user.badUsername;
```

Comment:
Для повышения читаемости кода, при использовании тернарных операторов, можно использовать переменные*:
```suggestion
    const isAgeOk = user.age >= 18 && user.age <= 35;
    const isAdmin = user.isAdmin;
    const paid = user.paid;
    const blocked = user.blocked;
    const badUsername = user.badUsername;
    return isAgeOk && (isAdmin || paid && !blocked && !badUsername) ? true : false;
```
Вместо блока с переменными еще можно использовать деструкцию объекта**:
```
function hasAccessTern({ age, isAdmin, paid, blocked, badUsername }) {
```

*не обязательно, но лучше делать тернарники максимально информативными
**не обязательно, но тоже повышает читабельность кода